core/station/api/spec.did

```diff
@@ -2583,7 +2583,10 @@ type SystemUpgraderInput = variant {
   // An existing upgrader canister.
   Id : principal;
   // Creates and deploys a new canister.
-  WasmModule : blob;
+  Deploy : record {
+    wasm_module : blob;
+    initial_cycles : nat;
```

Comment:
maybe make it optional and assume a value by default (e.g. 1.5T)?

Reply:
this value must also cover code installation and thus 1.5T might not be enough on the fiduciary subnet; why would you prefer an option? for the sake of disaster recovery?

Reply:
As discussed in a meeting, I've made the upgrader initial cycles optional with a default value of 2.5T.